docs/tempo/website/setup/_index.md

```diff
@@ -1,40 +1,38 @@
 ---
-title: Set up a Tempo cluster
-menuTitle: Set up a Tempo cluster
-description: Learn how to set up a Tempo cluster and visualize data
+title: Set up a Tempo server or cluster
+menuTitle: Set up a Tempo server or cluster
+description: Learn how to set up a Tempo server or cluster and visualize data
 weight: 150
 ---
 
-# Set up a Tempo cluster
+# Set up a Tempo server or cluster
 
 Tempo is available as a pre-compiled binary, a Docker image, and as common OS-specific packaging.
 
-No matter which option you choose, setting up your Tempo cluster involves: 
+You can choose to install Tempo on a single system or as a cluster.
 
-1. Naming your cluster
-1. Deploying your cluster
-1. Testing your cluster
-
-This page highlights these steps; more detailed instructions are available on the procedures for deploying clusters.
+This page highlights these steps; more detailed instructions are available on the procedures for installing Tempo.
 
 ## Name your cluster
 
-No matter which deployment method you choose, you need to decide what to call your Tempo cluster.
+If you install Tempo in a cluster, you need to decide what to call it.
 A cluster name must meet the following criteria:
 
 - is 3 to 63 characters long
```

Comment:
where do these restrictions come from?

Reply:
They are actually from the [Enterprise Traces content](https://grafana.com/docs/enterprise-traces/latest/setup/). Is this incorrect for a Tempo cluster?

Reply:
I'll remove this section.